# Averaged values from the ActuatorStates topic
Header header
float32 arm
float32 bucket
float32 sled
